docusaurus.config.js

```diff
@@ -117,8 +117,39 @@ module.exports = {
                     docsPluginId: 'transport'
                 },
                 {
-                    to: '/learn/introduction',
-                    label: 'Unity Multiplayer Resources',
+                    to: 'about',
+                    label: 'Netcode Tools',
+                    position: 'left',
+                    docsPluginId: 'tools',
+                    className: 'nav-break',
+                    items: [
+                        {
+                            to: '/tools/tools-1-0-0-pre',
```

Comment:
This should probably link to documentation for 1-0-0

Reply:
That's specifically to the Release Notes. I can update the id name so it'll say 1-0-0, but it goes to the correct location. Actually... I'll start a new page for Release Notes 1.0.0+ and link that instead.

Reply:
The drop down for the Release Notes will point to the 1.0.0+ RN instead. The older RNs are still in the sidebar and I linked them in them in 1.0.0+ RNs.